settings-manager.js

```diff
@@ -0,0 +1,42 @@
+class SettingsManager extends EventTarget {
+  constructor() {
+    super();
+
+    this.addEventListener('gfxSettingsChanged', e => {
```

Comment:
The casing between event names is inconsistent. Also, we generally do not camelcase in event names, which is the standard for the web platform.

Reply:
noted